shared/test/unit/mocks/mock_l10n.js

```diff
@@ -8,7 +8,7 @@
     get: stringify,
 
     formatValue: function(id, args) {
-      return Promise.resolve(stringify(id, args));
+      return Promise.resolve(stringify(id, args || ''));
```

Comment:
I can't remember... I've re-run all the unit-tests and they pass w/o this change so it might have been related to something that was fixed in another bug (the lockscreen maybe?).